src/components/elements/popover/popover.tsx

```diff
@@ -29,6 +31,21 @@ export const Popover = (props: IGenericElementProps<IPopoverSchema>) => {
 		const Element = Icons[icon];
 		return <Element />;
 	};
+	const sanitizeOptions: IOptions = {
+		allowedTags: sanitizeHtml.defaults.allowedTags.concat(["img"]),
+		allowedAttributes: false,
+	};
```

Comment:
nitpick: can declare `sanitizeOptions` within `renderPopoverContent` since it's not referenced anywhere else